latex/device_class.tex

```diff
@@ -250,12 +250,14 @@ \subsubsection{Device information descriptors}
      {
       Returns the maximum size of memory object allocation in  bytes. The minimum value is max (1/4th of \codeinline{info::device::global_mem_size},128*1024*1024) if this SYCL \codeinline{device} is not of device type \codeinline{info::device_type::custom}.
      }
-
    \addInfoRow
      {info::device::image_support}
      {bool}
-     {Returns true if images are supported by this SYCL \codeinline{device} and false if they are not. }
-
+     {
+       Returns true if images are supported by this SYCL \codeinline{device} and
+       false if they are not. If this SYCL \codeinline{device} is a
+       \gls{host-device}, images must be supported.
```

Comment:
I do not know if this has to be restated here since already stated on line 1343-1344 but if so, it should be clarified by adding for example "and thus this function returns true."

Reply:
That's a good point, I've added this clarification.